src/components/dropdown/section-dropdown.tsx

```diff
@@ -2,6 +2,10 @@ import { BottomSheetSectionList, TouchableOpacity } from '@gorhom/bottom-sheet';
 import React, { FC, memo, useCallback } from 'react';
 import { FlatListProps, ListRenderItemInfo, Text, View } from 'react-native';
 
+import { TestIdProps } from 'src/interfaces/test-id.props';
+import { AnalyticsEventCategory } from 'src/utils/analytics/analytics-event.enum';
+import { useAnalytics } from 'src/utils/analytics/use-analytics.hook';
```

Comment:
Please avoid mixing relative and `src` imports from directories of the same level